src/example-relay/src/graphql/Schema.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import { GraphQLSchema, GraphQLObjectType } from 'graphql';
+
+import AllLocations from './queries/AllLocations';
+
+const RootQuery = new GraphQLObjectType({
+  name: 'RootQuery',
+  description: 'Root Query',
+  fields: {
+    allLocations: AllLocations,
```

Comment:
Do we need to keep this? Today we would name the query just `locations` I guess.

Reply:
Good idea, yeah, I was thinking to match the names of our current schema, but we can rather update the client when we do the switch 👍

Reply:
🚀